src/material/slider/slider-input.ts

```diff
@@ -275,22 +278,22 @@ export class MatSliderThumb implements _MatSliderThumb, OnDestroy, ControlValueA
    */
   protected _isControlInitialized = false;
 
-  private _platform = inject(Platform);
-
   constructor(...args: unknown[]);
 
   constructor() {
+    const renderer = inject(Renderer2);
+
     this._ngZone.runOutsideAngular(() => {
-      this._hostElement.addEventListener('pointerdown', this._onPointerDown.bind(this));
-      this._hostElement.addEventListener('pointermove', this._onPointerMove.bind(this));
-      this._hostElement.addEventListener('pointerup', this._onPointerUp.bind(this));
+      this._listenerCleanups = [
+        renderer.listen(this._hostElement, 'pointerdown', this._onPointerDown.bind(this)),
```

Comment:
Note that this one actually fixes a memory leak. Previously the events wouldn't have been removed, because `bind` returns a new function instead and for the `removeEventListener` call to work below, it has to be the same function.